includes/class-wc-install.php

```diff
@@ -310,6 +310,7 @@ public static function install() {
 		self::create_files();
 		self::maybe_create_pages();
 		self::maybe_set_activation_transients();
+		self::set_paypal_standard_load_eligibility();
```

Comment:
I'm not sure if this is the correct way to go about setting this flag on install. Let me know if there's a preferred way of doing this.